Mathlib/Tactic/Linter/Style.lean

```diff
@@ -68,4 +69,47 @@ initialize addLinter setOptionLinter
 
 end Style.setOption
 
+/-- The `check_declID` linter: if it emits a warning, then a declID is considered
+non-standard style.  -/
+register_option linter.style.check_declID : Bool := {
+  defValue := false
+  descr := "enable the `setOption` linter"
+}
+
+
+namespace Style.checkDeclID
+
+/-- Checks whether a given identifier name contains "__". -/
```

Comment:
This is not an identifier *name* (but syntax corresponding to one), right? Can you tweak the comment to make this clearer?